styles/styles.css

```
@@ -0,0 +1,610 @@
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 21px;
+}
+
+.catalog-filter-infrastructure,
+.catalog-type-of-hotels {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 24px;
+    text-transform: capitalize;
+}
+
+.infrastructure-list,
+.type-hotel {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 23px;
+}
+
+
+.button-official-site {
+    text-decoration: none;
+    color:#000000;
+    max-width: 115px;
+    margin-left: auto;
+}
+
+.field {
+    font-family: inherit;
+}
+
+h2 {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #000000;
+    display: flex;
+    justify-content: center;
+    margin-top: 69px;
+    margin-left: 290px;
+    margin-bottom: 120px;
+    margin-right: 290px;
+    padding: 0;
+}
+
+p {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    color: #333333;
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    margin-top: 25px;
+    margin-left: 274,5px;
+    margin-bottom: 69px;
+    margin-right: 274,5px;
+}
+
+
+h3 {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #000000;
+}
+
+.reservations {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #000000;
+}
+
+.reservations-text {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 26px;
+    text-align: center;
+    color: #333333;
+    text-transform: none;
+}
+
+.reservations-button {
+    font-family: inherit;
+    text-transform: uppercase;
+}
+
+.visually-hidden {
+    visibility: hidden;
+}
+
+.number-of-selected-hotels {
+    color:#FFFFFF;
+    background-color: #7DB54F;
+    text-decoration: none;
+    width: 20px;
+    height: 20px;
+    left: 20px;
+    top: 11px;
+    border-radius: 50%;
+    font-weight: 400px;
+    font-size: 10px;
+    line-height: 10px;
+    text-align: center;
+}
+
+.advantages-title-first,
+.advantages-title-fifth {
+    color:#FFFFFF;
+    text-align: center;
+}
+
+.advantages-description-first,
+.advantages-description-fifth {
+    color:#FFFFFF;
+    text-align: center;
+}
+
+.newsletter-text,
+.newsletter-text-catalog {
+    font-family: 'PT Sans';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 26px;
+    color: #FFFFFF;
+    text-align: center;
+    text-transform: none;
+}
+
+.newsletter-text-catalog {
+    color: #333333;
+}
```

```suggestion
.newsletter-text-catalog {
    color: #333333;
}
```

Можно от родителя.

Ведь цвет текста зависит от фона родителя, а не просто так сам по себе. Значит, и в селекторе мы можем это использовать.